Test/Unit/Aggregator/Category/CategoryCountAggregatorTest.php

```diff
@@ -0,0 +1,275 @@
+<?php declare(strict_types=1);
+
+namespace RunAsRoot\PrometheusExporter\Test\Unit\Aggregator\Shipment;
+
+use Magento\Catalog\Api\Data\CategoryInterface;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\DB\Adapter\AdapterInterface;
+use Magento\Framework\DB\Select;
+use Magento\Framework\DB\Sql\Expression;
+use Magento\Framework\DB\Sql\ExpressionFactory;
+use Magento\Framework\EntityManager\EntityMetadataInterface;
+use Magento\Framework\EntityManager\MetadataPool;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use RunAsRoot\PrometheusExporter\Aggregator\Category\CategoryCountAggregator;
+use RunAsRoot\PrometheusExporter\Service\UpdateMetricService;
+
+final class CategoryCountAggregatorTest extends TestCase
+{
+    private const METRIC_CODE = 'magento_catalog_category_count_total';
+    private const T_ATT = 'm2_eav_attribute';
+    private const T_CAT_ENT_INT = 'm2_catalog_category_entity_int';
```

Comment:
What are these abbreviations? Test? Table? I'd suggest to be more explicit here.